pyiceberg/io/fsspec.py

```diff
@@ -176,14 +186,50 @@ def _gs(properties: Properties) -> AbstractFileSystem:
 def _adlfs(properties: Properties) -> AbstractFileSystem:
     from adlfs import AzureBlobFileSystem
 
+    for property_name in properties:
+        if property_name.startswith(DEPRECATED_ADLFS_PREFIX):
+            deprecated(
+                deprecated_in="0.7.0",
+                removed_in="0.8.0",
+                help_message=f"The property {property_name} is deprecated. Please use properties that start with adls.",
+            )(lambda: None)()
```

Comment:
Just put a note here: a more elegant way to send deprecation message will be available in @ndrluis 's another PR: https://github.com/apache/iceberg-python/pull/963